resource_state_events_test.go

```diff
@@ -0,0 +1,186 @@
+package gitlab
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"testing"
+)
```

Comment:
Same about the ordering, so using `goimports` should make this look like:

```suggestion
import (
	"fmt"
	"net/http"
	"testing"
	
	"github.com/stretchr/testify/require"	
)
```

Rest of this looks fine 👍🏻